flink-scala/pom.xml

```diff
@@ -236,4 +230,23 @@ under the License.
 		</plugins>
 	</build>
 
+	<profiles>
+		<profile>
+			<id>scala-2.10</id>
+			<activation>
+				<property>
+					<!-- this is the default scala profile -->
+					<name>!scala-2.11.profile</name>
```

Comment:
Ah .. I need to drink more club mate before reviewing pull requests ;)
The profile is activated by default.
Then, you can also do
```xml
 <activation>
      <activeByDefault>true</activeByDefault>
    </activation>
```
and deactivate it with `-P!scala-2.10`.

Why didn't you make 2.10 the default and add a profile for 2.11 (which seems to be the special case)